### ISSUE TITLE
Hide pre-populated Posts list when Subscribing Without Commenting

### ISSUE THREAD

**Issue by raamdev** [websharks/comment-mail#203]
In https://github.com/websharks/comment-mail/issues/108#issuecomment-164202489 we hid the Add New Subscription form from the front-end, however one place it's not hidden (and cannot be hidden) is when Subscribing Without Commenting:

![2016-01-11_10-30-12](https://cloud.githubusercontent.com/assets/53005/12237761/51dde84a-b84e-11e5-8d7c-48ca9b768dd5.png)

That link opens the Add New Subscription page with the current post selected, however the reason we hid the Add New Subscription form in the first place was to prevent visitors from being able to use/see the pre-populated list of posts.

![2016-01-11_10-30-35](https://cloud.githubusercontent.com/assets/53005/12237820/a5c04944-b84e-11e5-8cc1-56a89ee3dd40.png)

What needs to happen is the Add New Subscription form needs to be updated so that it only shows the post being subscribed to (when Subscribing Without Commenting), and it should not be possible to change the Post or see a list of other posts.

**Comment by raamdev** [comment]
Referencing https://github.com/websharks/comment-mail/issues/199#issuecomment-170415300 where @RealDavidoff pointed this issue out.

**Comment by raamdev** [comment]
Related: #177

**Comment by RealDavidoff** [comment]
No, not only "Hide pre-populated Posts list when Subscribing Without Commenting" Raam, *always*, whether or not they comment.
Why show them posts in dropdown they haven't even read yet/ gained access rights? And why let them subscribe to comment replies to such posts?
Doesn't make sense!

ANY subscription should only be possible to what they've just read. 
It would never cross my mind to subscribe to some or all of Webshark's articles that I haven't even seen yet! (and, maybe, I wouldn't be interested in...). The idea is insane.

**Comment by raamdev** [comment]
> always, whether or not they comment.

There's no other time they can see the Add Subscription Form other than when Subscribing Without Commenting.

**Comment by raamdev** [comment]
## Possible Short-Term Solution

Switch to Advanced template mode and update this section of code as shown below in the template for the "Add/Edit Form":

![2016-01-12_20-48-11](https://cloud.githubusercontent.com/assets/53005/12282686/f7d5fb46-b96d-11e5-80f5-482c2cbb6f94.png)

Here's that updated code as shown in the screenshot above:

```
<?php echo $form_fields->select_row(
	array(
		'name'                => 'post_id', 'options' => '', 'current_value' => $current_value_for('post_id'),
		'input_fallback_args' => array('type' => 'hidden', 'maxlength' => 20, 'other_attrs' => 'min="1" max="18446744073709551615"', 'placeholder' => '', 'current_value_empty_on_0' => TRUE),
	)); ?>
```

**Comment by RealDavidoff** [comment]
I adjusted the code like you said, and here's what happens now:
1) Clicking "subscribe without commenting" (which still is so mini only teens can read it!) opens new (should be same) page "add new subcription" with "Comment" dropdown prepopulated with "All comments/replies". 
I agree, when s.o. reaches this page (subscribe Without commenting), it is sensible to show them the whole list of comments, because SUCH person can only be interested either in ALL comments or in a SPECIFIC comment reply. Thus here, the present full dropdown and prepopulation is perfect!

2) However, clicking "manage my subscriptions" (which too is so mini only teens can read it!) opens new (should be same) page "my comment subscriptions", and if HERE you click "You may click here to create one" (say because you have none yet, lol) then you seem to be going to the same "add new subcription" page as before, but coming THIS WAY there seems to be a problem:
When s.o. clicks "Manage my subscriptions" (in general!, ie he is not interested in "subscribing to this particular article without commenting"), then the "Comment" dropdown is NOT prepopulated with "All comments/replies" (only grey in there), and the individual comments are NOT shown in the dropdown, I can only scroll through numbers here. First I thought "hmm, seemingly the SETTINGS "yes enable comment select menu options" is NOT applied on this (same) page when called from the different page. A bug?", but then I realized: 
The comment dropdown itself makes no sense here(!), or does it? The person who came THIS WAY wants to "manage subscriptions" IN GENERAL, so why show him a COMMENT dropdown? Which, if you WANT to show it, should respect the SETTINGS and list (ALL! OMG!) comments by name, not number scrolling. No user will know which number to scroll for here! Sorry! Well, I don't know, and I am academic. And with the (present) "explanation" saying "If empty, you'll be subscribed to all comments/replies; i.e. NOT to a specific comment." I get scared! "Oh, what?? I don't know which number to pick here, but if I leave field empty I'll be subscribed to ALL comments on this website??? Get me out of here quickly!!" ;-)

The only dropdown that *would* make sense here when user comes THIS WAY, is a dropdown of all the ARTICLES that he "has had yet/ has READ yet/ was sent a link to read them" (access right, in our case). But you can't know that! Thus, in my opinion the offer "You may click here to create one" is a dangerous subject! Not thought through yet. I understand it's nice to offer them *something* when they click "manage my subscriptions" but don't have one yet, however the present COMMENT dropdown is useless in this case, and an ARTICLE dropdown would be problematic for any well-established site (think: thousands of articles/pages/custom-post-types!) and would be suicide in our case!
The solution? Hmm, I would almost like to suggest: Offer them nothing here. Write a note (let us add a note): "No subscriptions at this time. You may come back here when you have subscribed to one or more particular articles".

3) When you then get "Submission accepted; nice work!", below you get "Add new subcription" and here again, same problem as under 2) at the end: I can only scroll through numbers here! Obvious, because your plugin cannot know at this point which of the "millions" of other comments to dropdown here, as no new article is "active". Hence, drop it, don't show any form at all when user has just gotten "Submission accepted; nice work!".

**Comment by raamdev** [comment]
@RealDavidoff Thank you for the feedback. We will work towards improving this.
